docs/core/tools/dotnet-test.md

```diff
@@ -71,6 +73,10 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 
   Prints out a short help for the command.
 
+- **`--interactive`**
+
+  Allows the command to stop and wait for user input or action. For example, to complete authentication. Available since .NET Core 3.0 SDK.
```

Comment:
I think the other articles you did that had an "Available since ... " clause used a different format. I think they were bold?

Reply:
Good catch, thanks.  Most of them had been done without bold, so I went back and removed bold styling from the bold ones to standardize the presentation.